eox_core/api/v1/views.py

```diff
@@ -138,6 +139,51 @@ def get(self, request, *args, **kwargs):
         return Response(response_data)
 
 
+class EdxappUserUpdater(UserQueryMixin, APIView):
+    """
+    Handles API requests to partially update users.
+    """
+    authentication_classes = (OAuth2Authentication, SessionAuthentication)
+    permission_classes = (EoxCoreAPIPermission,)
+    renderer_classes = (JSONRenderer, BrowsableAPIRenderer)
+
+    def patch(self, request, *args, **kwargs):
+        """
+        Partially update a user from edxapp. Not all the fields can be updated, just the ones thought as `safe`.
+
+        For example:
+
+        **Requests**:
+            PATCH <domain>/eox-core/api/v1/user/
+
+        **Request body**
+            {
+                "email"=<USER_EMAIL>,    (or username=<USERNAME>)
+                "is_active": true,
+                "password": "new password"
+            }
+
+        **Response values**
+            User serialized.
+        """
+        # Pop identification
+        data = request.data.copy()
+        query_params = {
+            "username": data.pop("username", None),
+            "email": data.pop("email", None),
+        }
+        query = self.get_user_query(request, query_params=query_params)
+        user = get_edxapp_user(**query)
+
+        serializer = WrittableEdxappUserSerializer(user, data=data, partial=True)
+        serializer.is_valid(raise_exception=True)
+        serializer.save()
+
+        admin_fields = getattr(settings, 'ACCOUNT_VISIBILITY_CONFIGURATION', {}).get('admin_fields', {})
```

Comment:
I am not familiar with this setting. Is it from this plugin? From django, from the edx-platform core?
What are you expecting this admin_fields to contain?

Reply:
I followed how get users returned the data:
https://github.com/eduNEXT/eox-core/blob/36823821634e49f34abf799e4bbb180b75f8f84d/eox_core/api/v1/views.py#L130

should I changed it?

Reply:
I don't know. Perhaps it makes sense to refactor that into a method that we understand better, but I think we have dragged this long enough.

Here is the commit that added this line. Just for context as it does not add much info:
https://github.com/eduNEXT/eox-core/commit/e9b6dd225d1b99fdf58ca61de8954366c75e597c